package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "evaluation-request-interception",
+  "version": "1.0.0",
+  "description": "Idea Pool TA tool evaluation project - Request interception for UI testing",
+  "main": "index.js",
+  "scripts": {
```

Comment:
please add `npx playwright install` as a postinstall script, as its execution is required before being able to run the tests.